jar/src/cljc/sixsq/slipstream/client/api/utils/http_utils.cljc

```diff
@@ -0,0 +1,16 @@
+(ns sixsq.slipstream.client.api.utils.http-utils)
+
+(def http-lib-insecure-key :kvlt.platform/insecure?)
+
+(defn- process-insecure
+  [req]
+  (if (contains? req :insecure?)
+    (-> req
+      (merge {http-lib-insecure-key (:insecure? req)})
```

Comment:
You might want to change the `merge` to `assoc` to avoid creating an extra map unnecessarily.